### DISCUSSION TITLE
Sentry IO with BTCPay Server

### DISCUSSION THREAD

**Discussion by SireWilliamson** [btcpayserver/btcpayserver#2808] [Technical Support]
I've been tasked by my employer to add sentry IO support to the BTCPay Server.
I've added the appropriate .NET Core packages, and am going to inject the necessary DSN code.
I'm just curious, has anyone else done this? 
Is this something that people would be interested in making a permanent part of the project? 
As more larger organisations adopt bit coin, more are going to want to monitor the server.
General thoughts? Suggestions?

**Comment by dennisreimann** [comment]
We are explicitly keeping any kind of telemetry out of the software. :D
